webpack/containers/DashboardContainer.js

```diff
@@ -54,12 +54,7 @@ class DashboardContainer extends SearchManagerComponent {
     this.setState(searchParameters);
   }
 
-  componentDidUpdate(prevProps, prevState) {
-    if(prevState.programFilter != undefined && (prevState.programFilter !== this.state.programFilter || prevState.systemFilter !== this.state.systemFilter)) {
-      this.props.fetchSearchResults(DASHBOARD_CONTEXT, this.currentSearchParameters());
-      this.props.setLastSearch(this.currentSearchParameters());
-    }
-
```

Comment:
Not sure if this can be removed, now the program and system filters aren't working (when clicked they don't trigger a search). If you hit search after setting some program and system filters it works but then "clear advanced filters" doesn't trigger a new search any more.